src/solvers/dg_simplices/mesh.jl

```diff
@@ -0,0 +1,57 @@
+# !!! warning "Experimental features"
+
+abstract type AbstractMeshData{Dim, ElemType} end
+
+"""
+    VertexMappedMesh{Dim, ElemType, Nboundaries, Tv, Ti} <: AbstractMeshData{Dim, ElemType}
+
+`VertexMappedMesh` describes a mesh which is constructed by an reference-to-physical 
+mapping which can be constructed using only the vertex positions. 
+
+Wraps `MeshData` and `boundary_faces` in a dispatchable mesh type.
+"""
+struct VertexMappedMesh{Dim, ElemType, Nboundaries, Tv, Ti} <: AbstractMeshData{Dim, ElemType}
+  md::MeshData{Dim, Tv, Ti}
+  boundary_faces::Dict{Symbol, Vector{Int}}
+end
+
+Base.ndims(::VertexMappedMesh{Dim}) where {Dim} = Dim
+
+"""
+  VertexMappedMesh(VXYZ::NTuple{Dim, Vector{Tv}}, EToV, rd::RefElemData;
+                   is_on_boundary = nothing,
+                   is_periodic::NTuple{Dim, Bool} = ntuple(_->false, Dim)) where {Dim, Tv}
```

Comment:
```suggestion
    VertexMappedMesh(VXYZ::NTuple{Dim, Vector{Tv}}, EToV, rd::RefElemData;
                     is_on_boundary = nothing,
                     is_periodic::NTuple{Dim, Bool} = ntuple(_->false, Dim)) where {Dim, Tv}
```
I think you need four spaces to mark this as code